frontend/packages/dev-console/src/components/import/pac/PacSection.tsx

```diff
@@ -0,0 +1,150 @@
+
+  React.useEffect(() => {
+    if (loaded && !!pac && pac.data['app-link']) {
+      setGithubAppAvailable(true);
+      setFieldValue('pac.repository.githubAppAvailable', true);
+    }
+  }, [pac, loaded, setFieldValue]);
+
+  React.useEffect(() => {
+    setFieldValue('pac.formType', 'strategy');
+    setFieldValue('pac.repository.gitUrl', url);
+    setFieldValue('pac.repository.name', recommendRepositoryName(url));
+    setFieldValue('pac.repository.gitProvider', type);
+  }, [setFieldValue, url, type]);
+
+  React.useEffect(() => {
+    if (repoAlreadyExists) {
+      setFieldError('pac.pacHasError', 'Repository with same URL already exists');
+    }
+  }, [setFieldError, repoAlreadyExists, errors]);
+
+  if (repoAlreadyExists) {
+    return (
+      <FormSection>
+        <Alert
+          className="odc-devfile-strategy-section__error-alert"
+          isInline
+          variant="danger"
+          title={t('devconsole~Import is not possible.')}
+        >
+          {t(
+            'Repository with same URL already exists. Please update the Repository URL or change the Build Strategy to continue.',
+          )}
+        </Alert>
+      </FormSection>
+    );
+  }
+
+  return (
+    <>
+      {pac ? (
+        <>
+          {githubAppAvailable && type === GitProvider.GITHUB ? (
+            <>
+              <br /> <br />
+              <FormSection>
+                <FormGroup fieldId="method">
+                  <Flex>
+                    <FlexItem span={3}>
+                      <Tile
+                        data-test="github"
+                        title={t('devconsole~Use GitHub App')}
+                        onClick={() =>
+                          setFieldValue('pac.repository.method', PacConfigurationTypes.GITHUB)
+                        }
+                        isSelected={values.pac.repository.method === PacConfigurationTypes.GITHUB}
+                      />
+                    </FlexItem>
+                    <FlexItem span={3}>
+                      <Tile
+                        data-test="webhook"
+                        title={t('devconsole~Setup a webhook')}
+                        onClick={() =>
+                          setFieldValue('pac.repository.method', PacConfigurationTypes.WEBHOOK)
+                        }
+                        isSelected={values.pac.repository.method === PacConfigurationTypes.WEBHOOK}
+                      />
+                    </FlexItem>
+                  </Flex>
+                </FormGroup>
+              </FormSection>
+              <br />
+              <FormSection>
+                {values.pac.repository.method === PacConfigurationTypes.GITHUB && (
+                  <GithubSection pac={pac} />
+                )}
+                {values.pac.repository.method === PacConfigurationTypes.WEBHOOK && (
+                  <WebhookSection pac={pac} />
+                )}
+              </FormSection>
+            </>
```

Comment:
@Lucifergene instead of duplicating `AdvancedConfigurations` code here I will suggest that you make a new component for this and use that in both the forms.